styleguide/_typography.scss

```diff
@@ -16,10 +16,10 @@ $system-fonts: Helvetica, Arial, sans-serif;
 // our typography
 @mixin kiln-copy {
   font-family: $system-fonts;
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
this prevents mobile safari (and supposedly other mobile browsers) from zooming in the page and breaking all our styles. also it looks better.